### ISSUE TITLE
Volume?

### ISSUE THREAD

**Issue by mickhansen** [spotify/docker-kafka#3]
Not being that experienced with kafka it would be nice to have a VOLUME exposed for easily setting up a data container.

**Comment by rucka** [comment]
+1

**Comment by lpicanco** [comment]
+1

**Comment by zero-** [comment]
+1

**Comment by shiny** [comment]
+1

**Comment by DaveVdE** [comment]
How easy is it to do this?

**Comment by mickhansen** [comment]
@DaveVdE I guess it's not actually strictly required since you can map directories without them being specifically exposed as VOLUME.

It would still be a nice additions for newcomers. Personally i use wurstmeister/kafka these days

**Comment by DaveVdE** [comment]
You're right, I was looking for a simple docker image to get a local development env going on but I was intimidated by the wurstmeister image, but I never got this one working. The wurstmeister/kafka image worked out ok.
